source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -1003,16 +1028,17 @@ let from_json_list source_json_list =
       name: string;
       message_format: string;
       code: int;
+      location: JsonAst.LocationWithPath.t;
     }
 
     let parse ~path json =
       json_string_member ~path "name" json
       >>= fun name ->
       json_string_member ~path "message_format" json
       >>= fun message_format ->
-      json_integer_member ~path "code" json
-      >>= fun code ->
-      validate_code_uniqueness ~path code >>= fun () -> Result.Ok { name; message_format; code }
+      json_integer_member_with_location ~path "code" json
```

Comment:
That means we use the location of the `code`. Shouldn't we use the location of the whole json object for the rule? I think this would make more sense if we want to use it latter in VSCode.

Reply:
Ah, rules can be long, and then again the error just pertains to the `code` of a particular rule being same as another one defined before. I can change this but I strongly wanna keep it like this :)

Reply:
sure